Gemfile

```diff
@@ -23,6 +23,7 @@ group :development, :test do
   gem 'pry-rails'
   gem 'rspec-rails'
   gem 'shoulda-matchers'
+  gem 'launchy'
```

Comment:
Why is this gem being added?

Reply:
It was added when i was trying to solve some issues and i forgot to delete.